devtools/readme.md

```diff
@@ -0,0 +1,198 @@
+<!---
+title='Devtools Day 1'
+time='120'
+facilitation=true
+[objectives]
+1="Inspect the DOM"
+2="Identify the values of the design tokens in the CSS"
+3="Alter the values of the CSS Variables using DevTools "
+4="Select the Grid Badge in the Elements panel"
+5="Use the Grid Editor to change layout"
+6="Run a Lighthouse audit"
+7="Identify and resolve the reported issues"
+8="Inspect the Accessibility Tree and compare it to the DOM"
++++
+-->
+
+# Inspector Gadget
+
+### Prep
+
+For this workshop you will need:
+
+- [ ] [Chrome](https://www.google.com/intl/en_uk/chrome/) with Devtools
+- [ ] A [ChatGPT account](https://chat.openai.com/auth/login)
+- [ ] This [Devtools Workshop page](http://cyf-workshops.netlify.app/devtools/)
+
+You've already used a little bit of Devtools in ITD when you used [Lighthouse](https://developer.chrome.com/docs/lighthouse/) to audit your code. But there's much much more! Devtools is an [entire IDE](https://developer.chrome.com/) running inside your browser and it has hundreds of ways to inspect, test, and develop your code.
+
+<!--
+{{<note type="tip" title="Explore">}} -->
+
+Each browser has its own Devtools. Download Safari, Chrome Canary, and Firefox Developer Edition to explore the differences.
+
+<!-- {{</note>}} -->
+
+## Inspect the DOM
+
+```objectives
+- [ ] Inspect the DOM
+- [ ] Identify the values of the design tokens in the CSS
+- [ ] Alter the values of the CSS Variables using DevTools
+```
+
+<!--
+{{<note type="exercise" title="Explore">}} -->
+
+In groups of no more than five.
+
+1.  Set a [timer for 10 minutes](https://www.google.com/search?q=timer+for+10+minutes).
```

Comment:
```suggestion
1.  Set a [whole class timer for 10 minutes](https://www.google.com/search?q=timer+for+10+minutes).
```
Just in case we end up with a scenario where every group sets their own timer for 10 minutes ( yes, this could happen 😆 )